.travis.yml

```diff
@@ -80,7 +80,7 @@ install:
   - phing recall -Dversion=$VERSION -Ddb.database=drupal -Ddb.user=lightning -Ddb.password=lightning -Ddb.host=127.0.0.1
   - cd docroot
   # Execute database updates, if there are any.
-  - drush updatedb --yes
+  - drupal update:execute
```

Comment:
This is necessary in order to work around the fact that Drush does the wrong thing during updates (i.e., it instantiates a DrupalKernel instead of UpdateKernel), which triggers bizarre bugs. I spent a day chasing this one down. drush-ops/drush#2611